features/cryptocell/FEATURE_CRYPTOCELL310/sha1_alt.h

```diff
@@ -33,6 +33,7 @@ extern "C" {
 typedef struct
 {
     CRYS_HASHUserContext_t crys_hash_ctx;
+    int                    is_cc_initiated;
```

Comment:
style: I think the word we are looking for here is "initialised" and not  "initiated".

Reply:
OK.